build.rs

```diff
@@ -0,0 +1,14 @@
+use std::env;
```

Comment:
In general I’m against build files unless they’re absolutely necessary; could you give some motivation for why we need this ? I really don’t want to add a build.rs

Reply:
The `parse_self` in `tests/archive.rs` requires to know the build artifact
path. The old method is incorrect. The build artifact could be
`target/debug`, `target/release`, `target/<target-triple>/debug`.
I find out that we could rely on `OUT_DIR` variable to know
exactly where the build artifact path is. But the `OUT_DIR` variable is 
not set if [there is no `build.rs`][1] script.

We could create an empty build.rs and parsing `OUT_DIR` in `parse_self`
test if you prefer.

[1]: https://doc.rust-lang.org/nightly/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates